File: swea/D3/10505.소득 불균형/10505.py
T = int(input())
for tc in range(1, T+1):
    n = int(input())
    arr = list(map(int, input().split()))

    avg = int(sum(arr) / n)
    cnt = 0
    for i in arr:
        if i <= avg:
            cnt += 1
    
    print(f'#{tc} {cnt}')

File: swea/D3/10505.소득 불균형/10505.cpp
#include <iostream>
#include <vector>

using namespace std;

int T;

int main(){
  cin >> T;
  for(int t = 1; t <= T; t++){
    int n;
    cin >> n;

    vector<int> arr(n);
    
    int avg = 0;
    for(int i = 0; i < n; i++){
      cin >> arr[i];
      avg += arr[i];
    }
    avg /= n;

    int cnt = 0;
    for(auto x : arr){
      if(x <= avg) cnt++;
    }

    cout << '#' << t << ' ' << cnt << endl;
  }
}